pages/windows/reg-load.md

```diff
@@ -0,0 +1,8 @@
+# reg load
+
+> Load saved sub keys into a different sub key in the registry.
+> This is intended for troubleshooting and temporary keys.
+
+- Load a specified key from a backup file:
+
+`reg load {{key_name}} {{path/to/file}}`
```

Comment:
So is this the key name to load it into, or the key name to load selectively from the file?

Reply:
The key name to load it into.

Reply:
Right. Perhaps `Load a backup file into the specified key` or something might aid clarity here?

Reply:
Yes, that makes quite a bit more sense.